### PR TITLE
Add flycapture2 lib download rule for aarch64(e.g. Jetson TX1)

### PR DESCRIPTION
**Pull request by tongtybj**
I have add the new download link for aarch64 processor such as jetson TX1

### CODE REVIEW

File: pointgrey_camera_driver/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+We have to manually activate the camera driver by executing:
+/home/ubuntu/ros/kinetic/build/pointgrey_camera_driver/usr/flycap2-conf
```
Comment (mikepurvis) [inline pointgrey_camera_driver/README.md]: Please remove this file from the PR.

**Comment by mikepurvis** [comment]
Looks reasonable; I don't understand why Travis is failing to build with this change. Can you investigate?

**Comment by tongtybj** [comment]
@mikepurvis 
Sorry for the late response.
Fix the [bug](https://github.com/ros-drivers/pointgrey_camera_driver/pull/93/commits/ee04c75801392f3dec07b33c7b022ec0064c2fad#diff-f0cb7239ba2c37168d471afd265fa433R30).

Please check

**Comment by mikepurvis** [comment]
Looks great, thanks!

**Pull request merged**